source/Character.hx

```diff
@@ -424,6 +414,47 @@ class Character extends FlxSprite
 		}
 	}
 
+	function parseDataFile()
+	{
+		Debug.logInfo('Generating character (${curCharacter}) from JSON data...');
+
+		// Load the data from JSON and cast it to a struct we can easily read.
+		var jsonData = Paths.loadJSON('characters/${curCharacter}');
+		if (jsonData == null)
+		{
+			Debug.logError('Failed to parse JSON data for character ${curCharacter}');
+			return;
+		}
+
+		var data:CharacterData = cast jsonData;
+
+		var tex:FlxAtlasFrames = Paths.getSparrowAtlas(data.asset, 'shared');
+		frames = tex;
+
+		for (anim in data.animations)
+		{
+			var frameRate = anim.frameRate == null ? 24 : anim.frameRate;
+			var looped = anim.looped == null ? false : anim.looped;
+			var flipX = anim.flipX == null ? false : anim.looped;
+			var flipY = anim.flipY == null ? false : anim.looped;
```

Comment:
Shouldn't these be
```haxe
var flipX = anim.flipX == null ? false : anim.flipX;
var flipY = anim.flipY == null ? false : anim.flipY;
```

Reply:
That is correct, I have resolved this issue just now.